optoforce/src/optoforce/optoforce.py

```diff
@@ -91,6 +88,17 @@ def __init__(self):
         elif self._sensor_type == self._OPTOFORCE_TYPE_64:
             self._nb_sensors = 1
             self._nb_axis = 6
+            
+        self._scale = rospy.get_param("~scale")
+
+        if len(self._scale) != self._nb_sensors:
+            raise ValueError("The number of sensors and scale factor vectors given doesn't match.") 
+        else:
+            for x in range(self._nb_sensors):
+                if len(self._scale[x]) != self._nb_axis:
+                    print(self._nb_axis)
+                    print(self._scale[x])
+                    raise ValueError("Number of axis and scaling factors given doesn't match.") 
```

Comment:
Maybe you can concatenate the numbers in the Exception's message instead of printing them ?